lib/ocpp/v201/smart_charging.cpp

```diff
@@ -802,4 +804,26 @@ SmartChargingHandler::handle_external_limits_changed(const std::variant<Constant
     return request;
 }
 
+std::pair<ClearedChargingLimitRequest, std::optional<TransactionEventRequest>>
+SmartChargingHandler::handle_external_limit_cleared(double percentage_delta, ChargingLimitSourceEnum source) const {
+    // K13.FR.02
+    ClearedChargingLimitRequest cleared_charging_limit_request = {};
+    cleared_charging_limit_request.chargingLimitSource = source;
+
+    std::optional<TransactionEventRequest> transaction_event_request = {};
+
+    const auto& limit_change_cv = ControllerComponentVariables::LimitChangeSignificance;
+    const float limit_change_significance = this->device_model->get_value<double>(limit_change_cv);
+
+    // K13.FR.03
+    if (percentage_delta > limit_change_significance) {
+        transaction_event_request = TransactionEventRequest{};
+        transaction_event_request->triggerReason = TriggerReasonEnum::ChargingRateChanged;
```

Comment:
We're missing a lot of the required fields for a TransactionEventRequest.

Reply:
Added some to the type there are question how to fill in the rest. I have a thread here https://lfenergy.zulipchat.com/#narrow/stream/417676-EVerest.3A-Cloud-communication/topic/K13.20-.20Release.20External.20Charging.20Limit.20Transaction.20Details/near/474664050